tests/fixtures/backends.py

```diff
@@ -22,10 +22,12 @@
 from pymongo import MongoClient
 from pymongo.errors import CollectionInvalid
 
+from ralph.backends.data.clickhouse import ClickHouseDataBackend
 from ralph.backends.data.fs import FSDataBackend, FSDataBackendSettings
 from ralph.backends.database.clickhouse import ClickHouseDatabase
 from ralph.backends.database.es import ESDatabase
 from ralph.backends.database.mongo import MongoDatabase
+from ralph.backends.lrs.clickhouse import ClickHouseLRSBackend
 from ralph.backends.storage.s3 import S3Storage
 from ralph.backends.storage.swift import SwiftStorage
 from ralph.conf import ClickhouseClientOptions, Settings, core_settings
```

Comment:
It seems it has not been removed from conf.py. Furthermore, the one from src/ralph/backends/data/clickhouse.py should be imported here.